scripts/qa_crowbarsetup.sh

```diff
@@ -4100,13 +4091,9 @@ function onadmin_reapply_openstack_proposals()
 
 function onadmin_prepare_crowbar_upgrade()
 {
-    if iscloudver 4minus ; then
-        complain 11 "This upgrade path is only supported for Cloud 5+"
-    else
-        # using the API, due to missing crowbar cli integration
-        # move nodes to upgrade mode
-        safely curl -s -X POST $crowbar_api_digest $crowbar_api/installer/upgrade/prepare.json
-    fi
+    # using the API, due to missing crowbar cli integration
+    # move nodes to upgrade mode
+    safely curl -s -X POST $crowbar_api_digest $crowbar_api/installer/upgrade/prepare.json
```

Comment:
If we remove Cloud 4 support, then it means we remove the ability to upgrade from 4 to 5, which is debatable in the first place as it's a valid jenkins job we have for Cloud 5.

However, if we remove that ability, then we can drop a whole codepath related to the old way of doing upgrades.